build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'maven'
 apply plugin: 'signing'
 
 group 'net.quedex'
-version '0.7.0-SNAPSHOT'
+version '0.8.0-SNAPSHOT'
```

Comment:
Version shouldn't be bumped. Please revert.

Reply:
OK